html/elements/script.json

```diff
@@ -548,6 +548,224 @@
+                },
+                "status": {
+                  "experimental": true,
+                  "standard_track": true,
+                  "deprecated": false
+                }
+              }
+            },
+            "referrerpolicy": {
+              "__compat": {
+                "description": "<code>referrer_policy</code> key",
+                "support": {
+                  "chrome": {
+                    "version_added": "111"
+                  },
+                  "chrome_android": "mirror",
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "ie": {
+                    "version_added": false
+                  },
+                  "oculus": "mirror",
+                  "opera": "mirror",
+                  "opera_android": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror",
+                  "samsunginternet_android": "mirror",
+                  "webview_android": "mirror"
+                },
+                "status": {
+                  "experimental": true,
+                  "standard_track": true,
+                  "deprecated": false
+                }
+              }
+            },
+            "requires": {
+              "__compat": {
+                "description": "<code>requires</code> key",
+                "support": {
+                  "chrome": {
+                    "version_added": "110"
+                  },
+                  "chrome_android": {
+                    "version_added": "103"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "ie": {
+                    "version_added": false
+                  },
+                  "oculus": "mirror",
+                  "opera": "mirror",
+                  "opera_android": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror",
+                  "samsunginternet_android": "mirror",
+                  "webview_android": "mirror"
+                },
+                "status": {
+                  "experimental": true,
+                  "standard_track": true,
+                  "deprecated": false
+                }
+              },
+              "anonymous-client-ip-when-cross-origin": {
+                "__compat": {
+                  "description": "<code>anonymous-client-ip-when-cross-origin</code>",
+                  "support": {
+                    "chrome": {
+                      "version_added": "110",
+                      "notes": "Chrome includes a browser-specific mechanism to provide an anonymous client IP."
```

Comment:
Is this behavior against the spec or otherwise incompatible? If not, I guess it belongs in the docs only and not in BCD?

"browser-specific mechanism" is also quite vague, so I'm not sure what value this note adds. To me it leaves the impression "Chrome does something weird here" and I guess that is not the impression you want to give.

Reply:
You are right, this isn't against the spec or anything, so such explanations just belong in the docs. I have removed the notes.